gnomad_qc/v3/create_release/prepare_vcf_data_release.py

```diff
@@ -822,7 +840,7 @@ def build_parameter_dict(
             "include_age_hists": True,
             "sample_sum_sets_and_pops": {"hgdp": HGDP_POPS, "tgp": TGP_POPS},
             "vcf_info_reorder": VCF_INFO_REORDER,
-            "ht": release_sites().ht(),
+            "ht": hl.read_table("gs://gnomad-vrs-io-finals/ht-outputs/release_0426_dmtest_v3.1.2-Full-ht-release-output-updated-schema-050523.ht"), #TODO: wait until the release HT to be updated to VEP 105 and created as a minor release
```

Comment:
gs://gnomad-vrs-io-finals/ was used due to requester pays issues writing to the typical gnomad bucket, so this path will change

Reply:
sorry to comment on a resolved ticket, but I think this got moved to 

`gs://gnomad/release/3.1.4/ht/genomes/gnomad.genomes.v3.1.4.sites.ht/`

if we're looking to formalize the code a bit more